app/src/common/shared/com/igalia/wolvic/ui/widgets/Windows.java

```diff
@@ -544,7 +544,7 @@ public boolean isInPrivateMode() {
 
     public boolean isVideoAvailable() {
         for (WindowWidget window: getCurrentWindows()) {
-            if (window.getSession().isVideoAvailable()) {
+            if (window.getSession().getActiveVideo() != null) {
```

Comment:
We may not need this change if we keep IsVideoAvailable, which seems clearer.

Reply:
It would be nice to rename `Session.getActiveVideo()` to `getActiveMedia()` and, for consistency, rename `Windows.isVideoAvailable()` to `hasActiveMedia()`.

The reason is that these methods handle objects of type `Media` which are not necessarily just videos.

Although that's probably too many changes for this one PR :slightly_smiling_face:

Reply:
Yeah I always have this in my backlog, but never find time to do it? Maybe @javifernandez ?